Example 2 (in-proc) breaks the code for Example 1

fabianuri opened the pull request:
readers should be aware that the code for example 2 breaks example 1.

fabianuri committed:
Example 2 (in-proc) breaks the code for Example 1 

readers should be aware that the code for example 2 breaks example 1.

mikematteson commented:
@laurenhughes  : 

PR review against the [PR checklist](https://review.docs.microsoft.com/windows-authoring-guide/managing-contributions/editorial-checklist?branch=master) is complete. Please review further for technical accuracy, and address the following issue:

FILE: **convert-app-service-in-process.md**

- :x: Requirement: *Markdown extensions must be used where appropriate and available. (For example, notes are coded using the [!NOTE] extension, not as plain text).*

  **If the PR is approved with the note intact, please use the appropriate Markdown extension syntax.**

fabianuri commented on windows-apps-src/launch-resume/convert-app-service-in-process.md:
```diff
@@ -42,6 +42,7 @@ Remove the `EntryPoint` attribute from the `<Application>` element because now [
 The second change is to move the service logic from its separate background task project into methods that can be called from **OnBackgroundActivated()**.
 
 Now your application can directly run your App Service. For example, in App.xaml.cs:
+NOTE: The code below is not the same as that example 1 (out-of-process service). The code below is just a sample.  For consistency, use the app service code from Example 1 instead of the code below.
```
[!NOTE]

fabianuri replied:
[!NOTE] The code below is different than the one provided for example 1 (out-of-process service). The code below is provided for illustration purposes only and should not be used as part of example 2 (in-process service).  To continue the article’s transition from example 1 (out-of-process service) into example 2 (in-process service) continue to use the code provided  for example 1 instead of the illustrative code below.

fabianuri committed:
Edited text again.

[!NOTE] The code below is different than that of example 1 (out-of-process service). The code below is new sample provided for illustration purposes in example 2 (in-process service).  Toi avoid breaking the code, continue to use the code provided earlier for example 1 instead of the code below.

fabianuri committed:
Improved suggested text.

[!NOTE] The code below is different than that of example 1 (out-of-process service). The code below is new sample provided for illustration purposes in example 2 (in-process service).  To avoid breaking example 1, continue to use the code provided  for example 1 instead of the new code below.

mikematteson commented:
@QuinnRadich : The article in this PR discusses Windows.ApplicationModel.AppService functionality, So would you please review this in Lauren's place? Note also that additional commits have been made since the original PR was submitted; looks like there are now two versions of the proposed text change, so one will need to come out. Thanks!

devtemp commented:
My sincere apologies. I want to help the readership but I don’t know how to use github. I don’t know where to click nor I know what you see at your end.  I’m really sorry for the mess. I’ll go online and make another attempt at the edition
		
		

		Get Outlook for iOS
	




On Wed, Jun 27, 2018 at 11:20 AM -0500, "Mike Matteson" <notifications@github.com> wrote:












@QuinnRadich : The article in this PR discusses Windows.ApplicationModel.AppService functionality, So would you please review this in Lauren's place? Note also that additional commits have been made since the original PR was submitted; looks like there are now two versions of the proposed text change, so one will need to come out. Thanks!



—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.

fabianuri committed:
Final edit

Third and last attempt at using github.  Pasting change below too. 
[!NOTE] The code below is different than the one provided for example 1 (out-of-process service). The code below is provided for illustration purposes only and should not be used as part of example 2 (in-process service).  To continue the article’s transition from example 1 (out-of-process service) into example 2 (in-process service) continue to use the code provided  for example 1 instead of the illustrative code below.

devtemp commented:
I took another stab at the edition. My issue is not knowing how github works. My final edit is below just in case. Thanks for your patience. 
[!NOTE] The code below is different than the one provided for example 1 (out-of-process service). The code below is provided for illustration purposes only and should not be used as part of example 2 (in-process service).  To continue the article’s transition from example 1 (out-of-process service) into example 2 (in-process service) continue to use the code provided  for example 1 instead of the illustrative code below.
		
		

		Get Outlook for iOS
	




On Wed, Jun 27, 2018 at 5:26 PM -0500, "Mike Matteson" <notifications@github.com> wrote:












@devtemp ; Thanks for your willingness to contribute to the docs and to try out GitHub at the same time. Certainly no apology is necessary! If you want to try committing another edit as you mentioned, that would be fine, or you can just stand by until one of our writers can have a look. Thanks again.


Cc: @QuinnRadich @laurenhughes



—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.

laurenhughes commented:
@devtemp, The code in example 1 (out-of-process) is meant to be modified using the information in the article to work with example 2 (in-process), resulting in an in-process app service. Do the code samples work for you if you modify XML in the app manifest and use the code in example 2?

devtemp commented:
The XML is OK.  The issue is *the abrupt* transition between example 1 to
example 2.  If a reader starting from example1 pastes the code from
example2 it breaks the code for example1.  If a new project for example2
should be created, the article should indicate this somewhere.  The article
for example1 has several incremental snippets, but upon reaching the code
snippet for example2, one soon realizes that it breaks the code for
example1.  One solution to the abrupt transition is to i) indicate users to
create a new UWP project for example 2 and ii) give steps to create such
new project (as done for example1).


On Tue, Jul 31, 2018 at 10:53 AM, Lauren Hughes <notifications@github.com>
wrote:

> @devtemp <https://github.com/devtemp>, The code in example 1
> (out-of-process) is meant to be modified using the information in the
> article to work with example 2 (in-process), resulting in an in-process app
> service. Do the code samples work for you if you modify XML in the app
> manifest and use the code in example 2?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/MicrosoftDocs/windows-uwp/pull/714#issuecomment-409310598>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AY1GC0tPuMmbyMyxKWxZpu2E1W77nmHKks5uMJmmgaJpZM4U2bk7>
> .
>

laurenhughes reviewed (approved):
Thanks, I've included the note! Updates should be available on Monday.
